db/api/controllers/handleErrors.js

```diff
@@ -1,7 +1,10 @@
-exports.handle404 = (req, res, next) => {
-  res.status(404).send({ msg: "Path not found" });
+exports.handleBadPaths = (err, req, res, next) => {
+  if (err.code === "22P02") {
+    res.status(400).send({ msg: "Path not found" });
```

Comment:
following from my previous comment - this is where the `msg` is not quite accurate